Documentation/ApiOverview/SiteHandling/SiteSettings.rst

```diff
@@ -1,56 +1,55 @@
-.. include:: /Includes.rst.txt
-.. index:: Site handling; Settings
-.. _sitehandling-settings:
+..  include:: /Includes.rst.txt
+..  index:: Site handling; Settings
+..  _sitehandling-settings:
 
 =============
 Site settings
 =============
 
-It is possible to define a `settings` block in a site's :file:`config.yaml` which
-can be accessed both in backend and frontend via the site object.
+It is possible to define a :yaml:`settings` block in a site's
+:file:`config.yaml` which can be accessed both in backend and frontend via the
+site object :php:`\TYPO3\CMS\core\Site\Entity\Site`.
 
-Additionally, these settings are available as "constants" in both TSConfig
-and TypoScript templates making it possible to configure for example site-wide
-storage PIDs which can be used in both frontend and backend.
+Additionally, these settings are available as "constants" in both
```

Comment:
why are the constants in `"` can we link them? Are we talking about TypoScript constants?

Reply:
Maybe as page TSconfig has no constants. I will remove the constants part.